### ISSUE TITLE
Jquery UI error

### ISSUE THREAD

**Issue by simondate** [sarveshwar-gavhane/adapt-multipleSlider#1]
I got this error in the web browser developer console when I tried to use the component:

jquery-ui.js:13475 Uncaught TypeError: this._addClass is not a function(…)

Seems to be an issue with this line https://github.com/sarveshwar-gavhane/adapt-multipleSlider/blob/master/js/adapt-multipleSlider.js#L39

Have you got this component working?

**Comment by sarveshwar-gavhane** [comment]
yes @simondate it has been working fine at our end in framework as well as authoring tool , can please tell me on which browser you have got this error ?

**Comment by simondate** [comment]
I've got this error on both Firefox and Chrome on MacOS 10.11.6

**Comment by sarveshwar-gavhane** [comment]
sorry @simondatewe don't have MacOS 10.11.6,  Had a quick build test on both Firefox and Chrome  MacOS 10.11.5 working fine .
